split_name.py

```diff
@@ -0,0 +1,26 @@
+import pandas as pd
+
+def inputspace(row):  # function for asking for user input for where the space should be

```

Comment:
Is there not a better solution than requiring human input?  If this is designed to iterate over every row, someone is going to be trapped for hours whenever you try to run your ETL.